2.rb

```diff
@@ -0,0 +1,7 @@
+def multiply(a,b)
+    a*b
```

Comment:
Add spaces between operation
```suggestion
    a * b
```